typescript/lib/config.ts

```diff
@@ -506,6 +525,37 @@ export class AIConfigRuntime implements AIConfig {
     delete this.metadata.models[modelName];
   }
 
+  /**
+   * Sets the model to use for all prompts by default in the AIConfig.
+   * @param modelName The name of the model to default to.
+   */
+  public setDefaultModel(modelName: string | undefined) {
+    if (modelName == null) {
+      delete this.metadata.default_model;
+      return;
```

Comment:
would it be beneficial to have a dedicated delete method?

Reply:
Discussed offline. Keeping this approach for now (consistent with `set env=` in CLI) -- we can add a delete method later if needed.